static/templates/confirm_dialog/confirm_moving_messages.hbs

```diff
@@ -0,0 +1,19 @@
+<p>
+	{{#if (and (eq messages_allowed_to_move 1) (eq messages_not_allowed_to_move 1))}}
+	{{#tr}}
+	You can't move all the messages in this topic, but you can move the latest message. Do you want to go ahead and do that? 1 message will remain in the current topic.
+	{{/tr}}
+	{{else if (eq messages_allowed_to_move 1)}}
+	{{#tr}}
+	You can't move all the messages in this topic, but you can move the latest message. Do you want to go ahead and do that? {{messages_not_allowed_to_move}} messages will remain in the current topic.
+	{{/tr}}
+	{{else if (eq messages_not_allowed_to_move 1)}}
+	{{#tr}}
+	You can't move all the messages in this topic, but you can move the latest {{messages_allowed_to_move}} messages. Do you want to go ahead and do that? 1 message will remain in the current topic.
+	{{/tr}}
+	{{else}}
+	{{#tr}}
+	You can't move all the messages in this topic, but you can move the latest {{messages_allowed_to_move}} messages. Do you want to go ahead and do that? {{messages_not_allowed_to_move}} messages will remain in the current topic.
+	{{/tr}}
+	{{/if}}
```

Comment:
I think there are too many if-else blocks here, but I am unable to figure out a better way to do it and also use "messages" or "message" correctly when required. Also, we cannot have "if" tags inside "tr" tags.